x-pack/qa/security-tools-tests/src/test/java/org/elasticsearch/xpack/security/authc/file/tool/UsersToolTests.java

```diff
@@ -70,7 +70,7 @@ public void setupHome() throws IOException {
         IOUtils.rm(homeDir);
         confDir = homeDir.resolve("config");
         Files.createDirectories(confDir);
-        hasher = inFipsJvm() ? randomFrom(Hasher.PBKDF2, Hasher.PBKDF2_1000)
+        hasher = inFipsJvm() ? randomFrom(Hasher.PBKDF2, Hasher.PBKDF2_1000, Hasher.PBKDF2_STRETCH)
```

Comment:
Technically we don't block the use of `PBKDF2_STRETCH` for non-FIPS clusters. So it seems to me that we should test it regardlessly, i.e. add it to the `randomFrom` list when `inFipsJvm()` is `false` as well.

Reply:
These are not meant to provide extensive test hashing coverage. We do that in HasherTests. Iirc this was added when I changed the behaviour of the cli tool to throw a useful error when in fips but using an algorithm that's not pbkdf2 based . 
I'm saying that in the sense that if we do that here ,then we might as well go and add the stretched variant as a possibility in all test cases where hashing a user pwd is part of the test setup. But I dont think we'd gain something by doing so